packages/backend/src/graphql/schema.graphql

```diff
@@ -241,6 +241,10 @@ type Mutation {
     input: CreateWalletAddressKeyInput!
   ): CreateWalletAddressKeyMutationResponse
 
+  createTenantSettings(
+    input: [CreateTenantSettingsInput!]!
```

Comment:
```suggestion
    input: CreateTenantSettingsInput!
```
and then just have the input have a `settings` list (like you have for the CreateTenantSettingsMutationResponse). Good for consistency and allows us to add additional fields to the input as well